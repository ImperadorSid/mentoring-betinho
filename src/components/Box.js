import styled from 'styled-components'
import {
  compose,
  position,
  space,
  color,
  layout,
  border,
  flexbox,
} from 'styled-system'

const Box = styled.div(
  {},
  compose(position, layout, space, color, border, flexbox)
)

export default Box
