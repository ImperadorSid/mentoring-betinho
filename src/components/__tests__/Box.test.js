import { render } from '@testing-library/react'
import Box from '../Box'

describe('<Box />', () => {
  it('renders component unchanged', () => {
    const { container } = render(<Box />)

    expect(container).toMatchSnapshot()
  })
})
