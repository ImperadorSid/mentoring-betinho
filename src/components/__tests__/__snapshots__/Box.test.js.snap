// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> renders component unchanged 1`] = `
<div>
  <div
    class="sc-bdvvtL"
  />
</div>
`;
